src/tree/registries/azure/azureRegistryProvider.ts

```diff
@@ -15,5 +17,7 @@ export const azureRegistryProvider: IRegistryProvider = {
     api: RegistryApi.DockerV2,
     onlyOneAllowed: true,
     connectWizardOptions: undefined,
-    treeItemType: AzureAccountTreeItem
+    treeItemFactory: (parent: AzExtParentTreeItem, cachedProvider: ICachedRegistryProvider) => new AzureAccountTreeItem(parent, cachedProvider),
+    persistAuth: undefined,
```

Comment:
Nit: the `undefined` properties can just be omitted?

Reply:
I guess I favor being explicit here so it's clear that they aren't forgotten but intentionally left out...